.gitlab-ci.yml

```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
     - master
     - frankfurt
   script:
-    - kubectl_1.13 apply -f frankfurt/nucleus-dev
-    - kubectl_1.13 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
-    - kubectl_1.13 apply -f frankfurt/nucleus-stage
+    - kubectl_1.11 apply -f frankfurt/nucleus-dev
+    - kubectl_1.11 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
+
+deploy frankfurt stage:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
```

Comment:
Do we want changes to `master` to update both dev and stage environments?

Reply:
My thinking is that a push to the `master` branch of this config repo should trigger a `kubectl apply` across all environments in parallel, hence the separate jobs. For troubleshooting purposes, I've also added some optional branches we could use to target specific deployments.